test/extended/images/mysql_replica.go

```diff
@@ -27,8 +27,8 @@ var (
 		{
 			"5.5",
 			"https://raw.githubusercontent.com/openshift/mysql/master/5.5/examples/replica/mysql_replica.json",
-			// TODO: Investigate why this is broken.
-			true,
+			// NOTE: Set to true in case of flakes.
```

Comment:
make this a `TODO: This is set to false because it is causing flakes`. Also open a follow up issue to track this.